integration-tests/tests/resource-observer/resource-obs.spec.js

```diff
@@ -0,0 +1,50 @@
+/*
+Copyright 2020 Splunk Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+module.exports = {
+  'should report resource loads happening after page load': async function(browser) {
+    browser.globals.clearReceivedSpans();
+    await browser.url(browser.globals.getUrl('/resource-observer/resources.ejs'));
+
+    const imageSpan = await browser.globals.findSpan(
+      span => span.tags['http.url'] && span.tags['http.url'].endsWith('splunk-white-black.png')
+    );
+
+    await browser.assert.ok(imageSpan, 'Image span not found.');
```

Comment:
AFAIK the string which is the second parameter is not a failure message, but rather a name or a description of the assertion.